mod/wikirate/set/type_plus_right/metric_value/source.rb

```diff
@@ -1,7 +1,6 @@
 format :html do
   view :editor do |args|
-    
-    if card.new?
+    if card.new? && !(Env.params['slot'] && Env.params['slot']['source'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.